sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -624,6 +630,52 @@ class EthService(
     }
   }
 
+  @RpcMethod("debug_traceCall")
+  @RpcOptionalParameters(1)
+  def traceCall(tx: TransactionArgs, tag: String, config: TraceOptions): Any = {
+
+    applyOnAccountView { nodeView =>
+      // retrieve block hash from input
+      val hash = getBlockHashByTag(nodeView, tag)
+
+      // get block to trace
+      val (block, blockInfo) = getBlockById(nodeView, bytesToId(hash.toBytes))
+
+      // get state at previous block
+      getStateViewAtTag(nodeView, (blockInfo.height - 1).toString) { (tagStateView, blockContext) =>
```

Comment:
Why do we need to go to the parent block here?

Reply:
typo, fixed